src/components/appNavigation/actionBar/hooks/useActionLink.js

```diff
@@ -21,6 +41,57 @@ const addExtraProps = (value, key) => ({
   icon: value.icon || key,
 })
 
+export const socials = mapValues(
+  {
+    x: {
+      icon: XIcon,
+      url: 'https://ubi.gd/twitter',
+      event: GOTO_X,
+    },
+    tg: {
+      icon: TgIcon,
+      url: 'https://t.me/GoodDollarX',
+      event: GOTO_TG,
+    },
+    dsc: {
```

Comment:
use a single list of actions, remove icons from here. icons should be part of the component

Reply:
Done